risc0/zkvm/Cargo.toml

```diff
@@ -34,6 +34,7 @@ borsh = { version = "1.5", default-features = false, features = ["derive"] }
 bytemuck = { version = "1.13", features = ["extern_crate_alloc"] }
 getrandom = { version = "0.2", features = ["custom"] }
 hex = { version = "0.4.3", default-features = false, features = ["alloc"] }
+redis = { version = "0.27.2", default-features = false, features = ["tls-native-tls"]}
```

Comment:
Would be good to make this enabled by a feature flag

Reply:
Great advice, cleaned up nicely all the lock files: [75d1fb8](https://github.com/risc0/risc0/pull/2407/commits/75d1fb84a670402dda2cd67f0b84c79af759a172)